demo/build/veraison.mk

```diff
@@ -1,10 +1,17 @@
-
 VERAISON_ROOT := .veraison-services
 
+# The command line can override VERAISON_BRANCH to point to a branch other than
+# main
+VERAISON_BRANCH ?= main
```

Comment:
Could we tie this to a particular commit? Otherwise it's not strictly reproducible.

Reply:
true dat.  I'd like to defer this though cos I sense there may be multiple ways to do this (e.g., using hooks from the veraison docker machinery) which I'd like to survey first.